ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -40,21 +48,15 @@ const renderFirstNotification = (notification, idRefMap) => {
       key={`whats-new-popop-notificatiion-${id}`}
       ref={idRefMap[id]}
     >
-      {image && (
-        <img
-          className="whats-new-popup__notification-image"
-          src={image.src}
-          height={image.height}
-          width={image.width}
-        />
-      )}
+      {image && !image.placeBelowDescription && imageComponent}
```

Comment:
The leading check for `image` shouldn't be necessary here given that it's already checked for here: https://github.com/MetaMask/metamask-extension/pull/10964/files#diff-8a1b237ca446aed86e9d755decb615e0f1b7b3034bf4920781bc494a81aff657R35

Reply:
Addressed in f1579b911